pkg/trustengine/trustengine.go

```diff
@@ -0,0 +1,83 @@
+package trustengine
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	piperhttp "github.com/SAP/jenkins-library/pkg/http"
+	"github.com/SAP/jenkins-library/pkg/log"
+)
+
+type Secret struct {
+	Token  string `json:"sonar,omitempty"`
```

Comment:
Is the json mapping correct (for now)?

Reply:
No, and I removed it, because currently the response's format makes it impossible for us to directly match it with a struct.